### DISCUSSION TITLE
Can I calculate velocity for each of the two conditions?

### DISCUSSION THREAD

**Discussion by YeEun8** [theislab/scvelo#691] [Q&A]
Hello, 
thank you for making such a great tool.
I analyzed single cell rna-seq data of two conditions of control / treatment with Seurat.
And I did a velocity analysis with total cells, but could the velocity change depending on each condition?

For example, when the velocity is calculated only with the control cells and only with the treatment cells, is there a possibility that the directionality shown in the total cells will be significantly different in each conditions?
The umap value is used by taking the value calculated from Seurat as total cells.

And is it the right analysis to compare the difference in velocity according to each condition by analyzing like this?

**Comment by WeilerP** [comment]
@YeEun8, yes, the velocities can change and I'd assume they will. Velocities tell you something about cellular differentiation. Consequently, by studying treatment/control independently, you might be able to identify different driver genes or differentiation pathways, for example.
The result of your RNA velocity analysis, however, always depends on the quality of the data, *i.e.*, if it offers any info to infer model parameters confidently.
